.github/workflows/fuzzing.yml

```diff
@@ -2,6 +2,8 @@ name: Fuzzer execution
 
 on:
   workflow_dispatch:
+  schedule:
+    - cron: '0 3 * * *'
```

Comment:
Can we also leave a comment with the frequency, similar to what we have here https://github.com/Azure/azure-osconfig/blob/d29a9ac7bd7569abef2de58de4c1ebd843370a61/.github/workflows/insiders-fast.yml#L6

Reply:
Added a comment